setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
```

Comment:
Would have been better if you kept with the use of defining dependencies within the requirements.txt file. You could find the [sample setup.py here](https://github.com/HTTP-APIs/python-hydra-agent/blob/master/setup.py)

Reply:
Which dependencies? I don't think hydra-python-core has any.

Reply:
I meant to point to the use of find_packages() 😅